daemon/money/money.go

```diff
@@ -0,0 +1,16 @@
+package money
+
+import (
+	"github.com/40acres/40swap/daemon/database/models"
+	"github.com/shopspring/decimal"
+)
+
+type Money int64
+
+func NewFromBtc(amount decimal.Decimal, chain models.Chain) Money {
+	return Money(amount.Mul(decimal.NewFromInt(1e8)).IntPart())
+}
+
+func (m Money) ToBtc() decimal.Decimal {
+	return decimal.NewFromUint64(uint64(m)).Div(decimal.NewFromInt(1e8))
```

Comment:
`NewFromInt` receives int64 already
```suggestion
	return decimal.NewFromInt(m).Div(decimal.NewFromInt(1e8))
```
although maybe is better to move Money to uint64

Reply:
Done